ci/terraform/sandpit.tfvars

```diff
@@ -15,10 +16,16 @@ support_authorize_controller  = "1"
 support_account_interventions = "1"
 support_2fa_b4_password_reset = "1"
 
+
 frontend_task_definition_cpu     = 256
 frontend_task_definition_memory  = 512
 frontend_auto_scaling_v2_enabled = true
 
+#cloudfront varaibles
+cloudfront_auth_frontend_enabled    = true
+auth_origin_cloakingheader          = "dsxUBpHUdcOYUBICSm3LuxA4V6yKGkz0"
+previous_auth_origin_cloakingheader = "dsxUBpHUdcOYUBICSm3LuxA4V6yKGkz0"
```

Comment:
These should be considered secret

Reply:
how do we mange secret for env which are deployed locally ? I can implemented that

Reply:
Ah this repo doesn't have a read secrets script in the deployment like api 😞